apps/openchallenges/db-update/update_db_csv.py

```diff
@@ -216,18 +222,20 @@ def main(gc):
     categories = get_challenge_categories(wks)
     output_csv(categories, "categories.csv", output_folder=CHALLENGE_FOLDER)
 
-    edam_annotations = get_edam_annotations(wks)
-    output_csv(edam_annotations, "challenge_data_edam.csv", output_folder=CHALLENGE_FOLDER)
-
     organizations = get_organization_data(wks)
     output_csv(organizations, "organizations.csv", output_folder=ORGANIZATION_FOLDER)
 
     challenges, incentives, sub_types = get_challenge_data(wks)
     output_csv(challenges, "challenges.csv", output_folder=CHALLENGE_FOLDER)
-    output_csv(incentives, "incentives.csv", 
-               output_folder=CHALLENGE_FOLDER, print_row=True)
-    output_csv(sub_types, "submission_types.csv",
-               output_folder=CHALLENGE_FOLDER, print_row=True)
+    output_csv(
+        incentives, "incentives.csv", output_folder=CHALLENGE_FOLDER, print_row=True
+    )
+    output_csv(
+        sub_types,
+        "submission_types.csv",
+        output_folder=CHALLENGE_FOLDER,
+        print_row=True,
+    )
```

Comment:
Should we go ahead and update this dumping script to include the dump file for `input_data_type.csv`?  Or we can save it for another PR.

Reply:
@vpchung Could you update this file as part of the present PR?

Reply:
Done ✅